data/migrations/20180907082952_projects.js

```diff
@@ -1,12 +1,14 @@
 
 exports.up = function(knex, Promise) {
    return knex.schema.createTable('projects', function(tbl) {
-      tbl.increments();
+      tbl
+         .increments();
       tbl
          .string('project')
          .notNullable()
          .unique('project');
-      tbl.string('description');
```

Comment:
Keeping things consistent is very important, nice! Personally, I'd start on the same line, like so
```
tbl.string('description');
```
but as long as it's **readable** and **consistent** is the most important 👍